net/mwan3/files/etc/hotplug.d/iface/15-mwan3

```diff
@@ -74,7 +77,9 @@ case "$ACTION" in
 	                mwan3_set_general_rules
 			[ "$status" = "online" ] && mwan3_set_policies_iptables
 		fi
-		[ "$ACTION" = ifup ] && procd_running mwan3 "track_$INTERFACE" && procd_send_signal mwan3 "track_$INTERFACE" USR2
+		[ "$ACTION" = ifup ] &&
```

Comment:
This is a cosmetic change and should also go to a separate commit

Reply:
Removed for now.